lucet-runtime/lucet-runtime-internals/src/module/dl.rs

```diff
@@ -188,7 +265,11 @@ impl ModuleInternal for DlModule {
     }
 
     fn trap_manifest(&self) -> &[TrapManifestRecord] {
-        self.trap_manifest
+        &self.code_metadata.trap_manifest
```

Comment:
Also not very sold on exposing `trap_manifest` here since it's moved, but if this reads fine, cool